NGS2_WITNESS_Cycle1_confirmatory_exp1.R

```diff
@@ -0,0 +1,73 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. in March 2017
+
+# Load data for analysis
+exp1_cooperation <- read.csv('NGS2-Cycle1-Experiment1/cooperation_exp1.csv',
+                             header = TRUE, sep = ',')
+exp1_rewire <- read.csv('NGS2-Cycle1-Experiment1/rewire_exp1.csv', header = TRUE,
+                        sep = ',')
+
+### Show stopper
+
+print("ATENTION!! Need to make sure the exp1_cooperation dataset includes the following variables: sessionnum, condition, decision..0.D.1.C.")
+
+#Hypothesis 4.1.1: Fixed network compositions reduce exp1_cooperation
+#Rand  et al. (2011) coeff = -0.186, p<.000
+
+exp1_cooperation.fixed<-subset(exp1_cooperation, condition=="Static")
+logit.fixed <- glm(decision..0.D.1.C.~ round_num, data = exp1_cooperation.fixed, family = "binomial")
+logit.multiwayvcov.fixed <- cluster.vcov(logit.fixed, cbind(exp1_cooperation.fixed$sessionnum, exp1_cooperation.fixed$playerid))
+coeftest(logit.fixed, logit.multiwayvcov.fixed)
+
+#Hypothesis 4.1.2: Randomly updating network compositions reduce exp1_cooperation
+#Rand  et al. (2011) coeff = -0.113, p<.000
+
+exp1_cooperation.random<-subset(exp1_cooperation, condition=="Random")
+logit.random <- glm(decision..0.D.1.C.~ round_num, data = exp1_cooperation.random, family = "binomial")
+logit.multiwayvcov.random <- cluster.vcov(logit.random, cbind(exp1_cooperation.random$sessionnum, exp1_cooperation.random$playerid))
+coeftest(logit.random, logit.multiwayvcov.random)
+
+#Hypothesis 4.1.3: Slowly updating strategic networks reduce exp1_cooperation
+#Rand  et al. (2011) coeff = -0.220, p=.013
+
+exp1_cooperation.viscous<-subset(exp1_cooperation, condition=="Viscous")
+logit.viscous <- glm(decision..0.D.1.C.~ round_num, data = exp1_cooperation.viscous, family = "binomial")
+logit.multiwayvcov.viscous <- cluster.vcov(logit.viscous, cbind(exp1_cooperation.viscous$sessionnum, exp1_cooperation.viscous$playerid))
+coeftest(logit.viscous, logit.multiwayvcov.viscous)
+
+#Hypothesis 4.1.4: Rapidly updating strategic networks support exp1_cooperation relative to all other conditions 
+#Rand  et al. (2011) coeff = 0.135, p = .006
+
+logit <- glm(decision..0.D.1.C.~ fluid_dummy*round_num, data = exp1_cooperation, family = "binomial")
+logit.multiwayvcov <- cluster.vcov(logit, cbind(exp1_cooperation$sessionnum, exp1_cooperation$playerid))
+coeftest(logit, logit.multiwayvcov)
+
+#Hypothesis 4.2.1	Rapidly updating strategic networks have greater network heterogeneity
+
+var <-aggregate(exp1_cooperation[,7], by=list(sessionnum=exp1_cooperation$sessionnum, fluid_dummy=exp1_cooperation$fluid_dummy), FUN=var)

```

Comment:
You might want to touch up the name of this variable as it's the same as the function `var()` that is used to create it. Not totally necessary, but could be helpful for others.

Reply:
Totally, thanks!